src/components/estimatedScoreWidget/locale/en.json

```diff
@@ -0,0 +1,19 @@
+{
+    "ESTIMATED_SCORE_WIDGET_DIRECTIVE": {
+        "TITLE": "SAT Estimated Score",
+        "PTS_TO_GO": "{{pts}} pts to go",
+        "YOUR_GOAL": "Your Goal: {{goal}}",
+        "GOAL_REACHED": "you've reached your goal!",
+        "COMPOSITE_SCORE": "Total Score:",
+        "GOAL_SCORE": "Goal Score:",
+        "EDIT_MY_GOALS": "Edit my goals",
+        "SUBJECT_0": "Math",
```

Comment:
the subjects name already defined int the app:
    "SUBJECTS":{
        "0": "math",
        "7": "verbal",
        "8": "essay"
    },